packages/shared_preferences/shared_preferences/lib/shared_preferences.dart

```diff
@@ -161,7 +161,7 @@ class SharedPreferences {
       } else {
         _preferenceCache[key] = value;
       }
-      return _store.setValue(valueType, prefixedKey, value);
+      return _store.setValue(valueType!, prefixedKey, value);
     }
   }
 
```

Comment:
Let's remove `commit` while making the breaking change.

Reply:
Not sure what you are talking about. The `commit` method is deprecated isn't it ?

Reply:
Right, `deprecated` means it shouldn't be used and will be removed in the future. Since it's been deprecated for a long time, and we're doing a breaking change for the plugin anway, it's a good time to actually remove it.